pkg/scheduler/internal/cache/cache.go

```diff
@@ -393,7 +393,7 @@ func (cache *schedulerCache) finishBinding(pod *v1.Pod, now time.Time) error {
 	cache.mu.RLock()
 	defer cache.mu.RUnlock()
 
-	klog.V(5).Infof("Finished binding for pod %v. Can be expired.", key)
+	klog.V(5).InfoS("Finished binding for pod. Can be expired.", key)
```

Comment:
According to the KEP sample `klog.InfoS("Pod status updated", "pod", "kubedns", "status", "ready")`, I think you have not followed the right pattern.

Reply:
Agree, this line should be:
```go
klog.V(5).InfoS("Finished binding for pod. Can be expired.", "pod", klog.KObj(pod), "key", key)
```